src/WSServer.cpp

```diff
@@ -101,6 +102,20 @@ void WSServer::notifyClients(const QJsonObject &obj)
     }
 }
 
+void WSServer::notifyGUI(const QString &message, bool &isGuiRunning)
+{
+    for (auto it = wsClients.begin(); it != wsClients.end(); ++it)
+    {
+        //Notify GUI
+        if (it.key()->resourceName().contains("localhost") && !(it.key()->origin().contains("extension")))
+        {
+            it.value()->sendJsonMessage(message);
```

Comment:
`isGuiRunning` should be set to true here, it's bad to let it be initialized to a value from the caller of this function.

Reply:
thanks, I forgot to add it there.